app/javascript/flavours/polyam/styles/components/columns.scss

```diff
@@ -238,7 +524,7 @@ $ui-header-height: 55px;
   align-items: center;
   gap: 5px;
   font-size: 16px;
-  padding: 15px;
+  padding: 13px;
```

Comment:
Upstream has larger icons, so this makes the items smaller, but I actually prefer that. Just something to keep in mind